crates/router/src/db/refund.rs

```diff
@@ -182,7 +182,7 @@ mod storage {
         //     merchant_id: &str,
         //     refund_id: &str,
         // ) -> CustomResult<Refund, errors::StorageError> {
-        //     let conn = pg_connection(self).await;
+        //     let conn = pg_connection_read(self).await;
```

Comment:
If you don't mind, please remove all commented out definitions of the `find_refund_by_payment_id_merchant_id_refund_id()` method.